operatorhub/kubernetes/config.yaml

```diff
@@ -0,0 +1,33 @@
+platform: "kubernetes"
+operator-packagename: "tektoncd-operator"
+image-substitutions:
+- image: ko://github.com/tektoncd/operator/cmd/kubernetes/operator:develContainer
```

Comment:
What `develContainer` means ?

Reply:
oops.. it was a debugging effort 😄 
removed 👍